opensyslog_syslog.py

```diff
@@ -142,6 +142,10 @@ def is_notification_needed(self, mac_address, ip_address, first_time_seen):
             case const.NOTIFY_CONNECT_EACH_TIME_WITH_MAX_PER_DAY_WITH_INTERMITTENT:
                 return (self.dhcp_ack_json[mac_address]["reconnect_count_per_day"] <= self.max_notify_count_per_device_per_day) or \
                     self.dhcp_ack_json[mac_address]["reconnect_count_per_day"] % 10 == 1
+            case const.NOTIFY_CONNECT_DEVICE_NOT_IN_LOOKUP_FILE:
+                if self.helper.lookup_device_name_from_csv(mac_address):
+                    return False
+                return True

```

Comment:
These 3 lines should be simplified to just one line to make it more readable please!
```
return not self.helper.lookup_device_name_from_csv(mac_address)
```

Reply:
Yes, I totally agree with that. I have made the changes.